app/models/lot.js

```diff
@@ -484,6 +483,18 @@ export default class LotModel extends Bookmarkable {
   get ownertypename() {
     return ownertypeLookup[this.ownertype];
   }
+
+  @computed('address')
+  get housenum() {
+    const match = this.address.match(/([0-9-]*)\s[0-9A-Za-z\s]*/);
```

Comment:
Does this regex allow for the house number to include a hyphen? I don't know regex well enough to discern, but we'll want to the house number to be able to accommodate hyphenated Queens addresses too. Ex: 36-45 31 STREET

Reply:
Nevermind, just checked the deploy preview and it's working!